### PR TITLE
refactor(printenv): remove indentation and use Console.print

### PR DESCRIPTION
**Pull request by Alizter**
Signed-off-by: Ali Caglayan <alizter@gmail.com>

### CODE REVIEW

File: bin/printenv.ml [hunk 1]
```diff
@@ -16,24 +16,17 @@ let dump sctx ~dir =
...
       in
-      if do_print then
-        let version =
-          Dune_lang.Syntax.greatest_supported_version Stanza.syntax
-        in
-        Dune_lang.Ast.add_loc sexp ~loc:Loc.none
-        |> Dune_lang.Cst.concrete |> List.singleton
-        |> Dune_lang.Format.pp_top_sexps ~version
-        |> Format.fprintf ppf "%a@?" Pp.to_fmt)
+      if do_print then Some (Dune_lang.pp sexp) else None)
```
Comment (emillon) [inline bin/printenv.ml:29]: You can use `Option.some_if`

File: bin/printenv.ml [hunk 2]
```diff
@@ -47,45 +40,45 @@ let term =
...
       in
-      Build_system.run_exn (fun () ->
-          let open Memo.O in
-          let+ res, _facts = Action_builder.run request Eager in
-          res)
-      >>| function
-      | [ (_, env) ] -> Format.printf "%a" (pp ~fields) env
-      | l ->
-        List.iter l ~f:(fun (name, env) ->
-            Format.printf "@[<v2>Environment for context %s:@,%a@]@."
```
Comment (rgrinberg) [inline bin/printenv.ml:87]: Why did  you remove the old boxes?

**Review by rgrinberg — CHANGES_REQUESTED** [review]

File: bin/printenv.ml (hunk 2 above)
Comment (Alizter) [reply bin/printenv.ml:87]: That's an oversight, I'll add the Pp version back in.

**Comment by Alizter** [comment]
Not really worth it.

**Pull request closed without merging**